### DISCUSSION TITLE
Issue with Starstruck achievement

### DISCUSSION THREAD

**Discussion by JNYH** [community/community#30130] [Other Feature Feedback, Questions, & Ideas]
I have quite a number of repos with many stars:

![image](https://user-images.githubusercontent.com/52286325/185677387-f29e2653-d77b-4dd1-9f6a-f09dea1db988.png)

However I'm only awarded with 1 Starstruck badge, is there a bug or is it normal? Do I have to do something?
Is there anyone else in the same situation?

![image](https://user-images.githubusercontent.com/52286325/185677573-9498d68f-c46e-42d9-b317-5089fe562db0.png)

**Comment by radar-blip** [comment]
You need a repo with 128 stars to get next level (bronze).
Here are some details:
https://github.com/drknzz/GitHub-Achievements

**Comment by jfullstackdev** [comment]
once the repo is selected, that repo should reach the next tier, take a look at Linus Torvalds, the creator of Linux OS

![image](https://user-images.githubusercontent.com/47092464/185750597-7c713233-c538-4ca9-a617-bdb3f66b6325.png)
